doc/classes/EditorUndoRedoManager.xml

```diff
@@ -80,6 +80,7 @@
 			<param index="0" name="name" type="String" />
 			<param index="1" name="merge_mode" type="int" enum="UndoRedo.MergeMode" default="0" />
 			<param index="2" name="custom_context" type="Object" default="null" />
+			<param index="3" name="backward_undo_ops" type="bool" default="false" />
```

Comment:
The new argument needs to be documented. You could maybe also mention in the class' description that undo is not reverse by default.